src/Compilers/CSharp/Portable/Symbols/MethodSymbol.cs

```diff
@@ -945,6 +960,52 @@ internal bool CalculateUseSiteDiagnostic(ref DiagnosticInfo result)
             return false;
         }
 
+#nullable enable
+        protected static UnmanagedCallersOnlyAttributeData DecodeUnmanagedCallersOnlyAttributeData(CSharpAttributeData attribute)
+        {
+            ImmutableHashSet<INamedTypeSymbolInternal>? callingConventionTypes = null;
+            bool isValid = true;
+            if (attribute.CommonNamedArguments is { IsDefaultOrEmpty: false } namedArgs)
+            {
+                foreach (var (key, value) in attribute.CommonNamedArguments)
+                {
+                    if (!UnmanagedCallersOnlyAttributeData.IsCallConvsTypedConstant(key, in value)
+                        || value.Values.IsDefaultOrEmpty)
+                    {
+                        continue;
```

Comment:
>continue [](start = 24, length = 8)

Are we setting `isValie = false;` for `[UnmanagedCallersOnly(CallConvs = new[] { typeof(ValidType) }, CallConvs = null)]`?